shaders/Monochrome.shader

```diff
@@ -0,0 +1,9 @@
+shader_type canvas_item;
+
+const vec4 rgbMultipliers = vec4(0.30, 0.59, 0.11, 1);
+
+void fragment() {
+    COLOR = texture(TEXTURE, UV) * rgbMultipliers;
+    float avg = (COLOR.r + COLOR.g + COLOR.b) / 3.0;
+    COLOR.rgb = vec3(avg);
```

Comment:
Isn't the math here still wrong? Shouldn't the RGB multipliers only add up to 1 at most, so taking the average on top of that seems not right to me?

Should it more be like: `luminosity = sum( texture(TEXTURE, UV) * rgbMultipliers)` (I'm not sure such "sum" function exists) and then `COLOR.rgb = vec3(luminosity)`